colaraz/cms/templates/settings_advanced.html

```diff
@@ -33,10 +34,13 @@
 <%block name="content">
 <div class="wrapper-mast wrapper">
   <header class="mast has-subtitle">
+    <%
+      index_url = reverse('course_handler', kwargs={'course_key_string': unicode(context_course.id)})
```

Comment:
What do you say?
```suggestion
      course_url = reverse('course_handler', kwargs={'course_key_string': unicode(context_course.id)})
```

Reply:
My first thought was also this but looking at the code of edx I prefered `index_url` to make the code consistent.

Kindly, have a look at it

https://github.com/colaraz/edx-platform/blob/develop/cms/templates/widgets/header.html#L22